src/libparmmgtypes.h

```diff
@@ -32,7 +32,7 @@
 #ifndef _LIBPARMMGTYPES_H
 #define _LIBPARMMGTYPES_H
 
-#include "mmg/mmg3d/libmmgtypes.h"
+#include "libmmgtypes.h"
```

Comment:
```suggestion
#include "mmg/common/libmmgtypes.h"
```
That's the single diff I have to apply to make everything work, last I checked.